src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/repository/LocationScheduleRepository.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.entity.LocationSchedule
+
+@Repository
+interface LocationScheduleRepository : JpaRepository<LocationSchedule, Long>
```

Comment:
Do we need this?  Would you not generally access a schedule via it's Aggregate root?

Reply:
Removed.